### ISSUE TITLE
loss can't down

### ISSUE THREAD

**Issue by JiaShengLiu111** [MG2033/MobileNet#6]
Thank for your share? I have a question is as follows:
The loss function cannot be reduced and all samples are predicted to be a certain category

I am looking forward for your answer, thank you very much!

**Comment by JiaShengLiu111** [comment]
Thank you any way! I already know where I am wrong. In your code, "tf.layers.batch_normalization" have been used, we should define the optimizer as follows:
          update_ops = tf.get_collection(tf.GraphKeys.UPDATE_OPS)
          train_op = optimizer.minimize(loss)
          train_op = tf.group([train_op, update_ops])
Reference：https://tensorflow.google.cn/api_docs/python/tf/layers/batch_normalization

**Comment by NowJzy** [comment]
> Thank you any way! I already know where I am wrong. In your code, "tf.layers.batch_normalization" have been used, we should define the optimizer as follows: update_ops = tf.get_collection(tf.GraphKeys.UPDATE_OPS) train_op = optimizer.minimize(loss) train_op = tf.group([train_op, update_ops]) Reference：https://tensorflow.google.cn/api_docs/python/tf/layers/batch_normalization

So how to change the source code?
